src/themes/teams/componentVariables.ts

```diff
@@ -22,4 +22,6 @@ export { default as ListItem } from './components/List/listItemVariables'
 
 export { default as Menu } from './components/Menu/menuVariables'
 
+export { default as MenuItem } from './components/Menu/menuVariables'
```

Comment:
I would consider this a hack.
But what is a clean solution?

- Split variables to menuVariables and menuItemVariables? Then you end up having several constants in both files + from designer's POV these are not separate but define one single Component
- Implement one variables as a wrapper for the other one?
- Naive copy&paste of the whole variables?